package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@keep-network/keep-core": ">1.8.0-dev <1.8.0-pre",
     "@keep-network/tbtc": ">1.1.2-dev <1.1.2-ropsten",
-    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#bf514ba",
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#a1384af",
```

Comment:
Let's update `yarn.lock` as well.

Reply:
Done in https://github.com/keep-network/coverage-pools/commit/e8bed549b7143437a0a70d19b3ba5cc82b2687ba.